doc/neps/nep-0036-fair-play.rst

```diff
@@ -0,0 +1,180 @@
+==================
+NEP 36 — Fair Play
+==================
+
+:Author: Stéfan van der Walt <stefanv@berkeley.edu>
+:Status: Draft
+:Type: Informational
+:Created: 2019-10-24
+:Resolution: Draft
+
+
+Abstract
+--------
+
+This document describes Rules of Play for engaging with the NumPy
+project as an external entity.  It discusses usage of the name NumPy,
+modified distributions of NumPy, and how to best communicate
+with the project when patched versions are deployed.
+
+Companies and developers will know after reading this NEP what kinds
+of behavior the community would like to see, and which we consider
+troublesome, bothersome, and unacceptable.
```

Comment:
```suggestion
We identify acceptable approaches from which both parties gain, and unacceptable ones that lead to confusion and wasted effort.
```

Reply:
I think there's more to it than this.  It is not only that there will be confusion and wasted effort, but that this will antagonize the developers.  And not only because they have to do more work (no-one here seems too scared of that), but because it goes against the philosophy of the project and does not feel like participation in good faith.

Reply:
Those clearly worded sentiments belong in the NEP;  "troublesome, bothersome, and unacceptable" is not the equivalent.

Using my wording a strawman, the NEP can say

> We identify acceptable approaches from which both parties gain, and unacceptable ones that lead to confusion and wasted effort and go against the philosophy of the project.

Insert your preferred wording; antagonism and bad faith can be folded in too. 

At any rate I'd urge against the _-some_ words. Their tone doesn't work in the intended way;  they look like they're waiting for "drat!".